openstates/tx/bills.py

```diff
@@ -259,30 +268,34 @@ def scrape_bill(self, session, history_url):
                     ctty = ctty.replace(r, "").strip()
                 extra['committees'] = ctty
 
-            bill.add_action(actor, action.findtext('description'),
-                            act_date, type=atype, **extra)
+            bill.add_action(
+                action.findtext('description'),
+                act_date,
+                chamber=actor,
+                classification=atype
+            )
```

Comment:
This used to pass a dictionary of extra info, `**extra`. If there isn't a way to just send along that extra info, then I need to find analogous [functions for passing that extra info](http://opencivicdata.readthedocs.io/en/latest/scrape/bills.html#overview) and remove any properties that no longer have a home.